test/org/zaproxy/zap/testutils/WebSocketTestUtils.java

```diff
@@ -39,16 +43,33 @@
     /**
      * Starts a WebSocket Server on specific hostname and port.
      */
-    public void startWebSocketServer(String hostname, int webSocketPort, int timeout) throws IOException {
+    public void startWebSocketServer(int webSocketPort, String hostname, int timeout) throws IOException {
```

Comment:
Any special reason why the parameters were swapped? Seems more natural to have the hostname first.

Reply:
I had problem with the following signatures:public void startWebSocketServer(String hostname, int timeout)

Reply:
Right, maybe drop `String hostname, int timeout`? We can always use the other constructor (`String hostname, int webSocketPort, int timeout`) to specify the timeout (with port as zero).